src/Symfony/Component/Mime/NamedAddress.php

```diff
@@ -39,4 +50,17 @@ public function toString(): string
     {
         return $this->getEncodedNamedAddress();
     }
+
+    public static function fromString(string $string): self
+    {
+        if (false === strpos($string, '<')) {
+            return new self($string, '');
+        }
+
+        if (!preg_match(self::FROM_STRING_PATTERN, $string, $matches)) {
+            throw new InvalidArgumentException(sprintf('Could not parse "%s" to a "NamedAddress" instance.', $string));
```

Comment:
Could be `sprintf('Could not parse "%s" to a "%s instance.', $string, static::class)`.